blocks/library/paragraph/index.js

```diff
@@ -81,6 +84,39 @@ class ParagraphBlock extends Component {
 		this.nodeRef = node;
 	}
 
+	setFontSize( value ) {
+		const { setAttributes } = this.props;
+		if ( fontSizeValues.includes( value ) ) {
```

Comment:
`Array#includes` is not polyfilled. This will error in IE11.

Reply:
You might try [`findKey`](https://lodash.com/docs/4.17.5#findKey) and an object of font size values to simplify this:

```js
const FONT_SIZES = {
	small: 14,
	// ...
};

// ...

setFontSize( fontSize ) {
	const size = findKey( FONT_SIZES, fontSize );
	let textClass;
	if ( size ) {
		textClass = `is-${ size }-text`;
	}

	this.props.setAttributes( { fontSize, textClass } );
}
```